apps/taskman/service.py

```diff
@@ -215,6 +215,8 @@ def _generate_task_data(self, flaw: Flaw):
             summary = flaw.title
         else:
             summary = f"{flaw.cve_id} {flaw.title}"
+        if len(summary) > 255:
```

Comment:
Instead of magic number use a constant with a descriptive name so it is clear what is the meaning of this